src/components/Button/Button.module.css

```diff
@@ -288,34 +286,30 @@
     color: var(--eds-theme-color-text-utility-disabled-primary);
     border-color: var(--eds-theme-color-border-disabled);
     background-color: var(--eds-theme-color-background-utility-default-no-emphasis);
-
-    pointer-events: none;
   }
 
   &.button--secondary.button--variant-inverse {
     color: var(--eds-theme-color-text-utility-inverse-disabled);
     border-color: var(--eds-theme-color-border-utility-inverse-disabled);
-
-    pointer-events: none;
   }
 
   &.button--tertiary.button--variant-default,
   &.button--tertiary.button--variant-critical,
   &.button--tertiary.button--variant-neutral {
     color: var(--eds-theme-color-text-utility-disabled-primary);
     background-color: var(--eds-theme-color-background-utility-default-no-emphasis);
-
-    pointer-events: none;
   }
 
   &.button--tertiary.button--variant-inverse {
     color: var(--eds-theme-color-text-utility-disabled-primary);
     border-color: transparent;
-
-    pointer-events: none;
   }
 }
 
+.button__disabled {
```

Comment:
this is to style a container when disabled. because `pointer-events` is also how the cursor gets styled, we cannot just add `cursor: not-allow